src/officehours_ui/templates/base.html

```diff
@@ -47,45 +54,60 @@
-                  <a class="dropdown-item" href="{% url 'manage' %}" style="background-color: white; color:#212529">
-                    Manage Queues
-                  </a>
-                  <a class="dropdown-item" href="{% url 'preferences' %}" style="background-color: white; color:#212529">
-                    Preferences
-                  </a>
-                  <form action="{% url 'oidc_logout' %}" method="post">
-                    {% csrf_token %}
-                    <input type="submit" value="Logout" style="background-color: white" class="dropdown-item">
-                  </form>
-                </div>
-              </li>
+                <form action="{% url 'oidc_logout' %}" method="post">
+                  {% csrf_token %}
+                  <input type="submit" value="Logout" style="background-color: white" class="dropdown-item">
+                </form>
+              </div>
+            </li>
             {% else %}
-              <a href="{{ LOGIN_URL }}" class="nav-link">Login</a>
+            <a href="{{ LOGIN_URL }}" class="nav-link">Login</a>
             {% endif %}
           </div>
         </div>
     </nav>
   </header>
-  
+
   {% for page in flatpages %}
-    {% if page.title == "User Comm Banner" %}
-      {{page.content|safe}}
-    {% endif %}
+  {% if page.title == "User Comm Banner" %}
+  {{page.content|safe}}
+  {% endif %}
   {% endfor %}
 
-	<div id="main-content" class="container" role="main">
+  <div id="main-content" class="container" role="main">
     {% block content %}{% endblock %}
     {% csrf_token %}
-	</div>
+  </div>
+
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+    integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+    crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+    integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+    integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+    crossorigin="anonymous"></script>
 
-  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
-  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+  <script>
+    $('#navbar-toggle').click(function () {
+      $(this).find('i').toggleClass('fas fa-bars text-light').toggleClass('fas fa-times');
```

Comment:
So, right now the X icon is black, which presents a color contrast and accessibility issue.
<img width="300" alt="Screenshot 2022-12-21 at 2 03 42 PM" src="https://user-images.githubusercontent.com/35741256/208987887-00e088ae-546b-44a9-b2ba-84c83c3940bf.png">

I think we want to keep the `text-light` icon instead of toggling. This line could also be simplified to just toggle the icon class and keep the color class. In other words:
```suggestion
      $(this).find('i').toggleClass('fa-bars').toggleClass('fa-times');
```
```

Reply:
Note that the switch to using `document` above also takes a similar approach, just toggling the icon class.